WordPressSecurityWhitePaper.html

```diff
@@ -104,14 +113,56 @@ <h4>A9 - Using Components with Known Vulnerabilities</h4>
 <p>If necessary, the core team may decide to fork or replace critical external components, such as when the SWFUpload library was officially replaced by the Plupload library in 3.5.2, and a secure fork of SWFUpload was made available by the security team<sup id="ref15"><a href="#footnote15">15</a></sup> for those plugins who continued to use SWFUpload in the short-term.</p>
 <h4>A10 - Unvalidated Redirects and Forwards</h4>
 <p>WordPress' internal access control and authentication system will protect against attempts to direct users to unwanted destinations or automatic redirects. This functionality is also made available to plugin developers via an API, <code>wp_safe_redirect()</code><sup id="ref16"><a href="#footnote16">16</a></sup>.</p>
+
+<h3>Threat Management</h3>
+
+<p>Threat management for the WordPress project covers two areas, the WordPress software and the WordPress.org infrastructure.</p>
+
+<h4>The WordPress Software</h4>
+
+<p>As a distributed, open-source, volunteer driven software project, the security of the source code needs to be maintained and monitored to protect it against malicious actors and unintentional regressions. Various functionality and processes are in place to help protect against these threats:</p>
+
+<ul>
+	<li>A meritocracy based trust process for core committers</li>
+	<li>Coding standards<sup id="ref30"><a href="#footnote30">30</a></sup> that aid legibility and consistency</li>
+	<li>A peer review process for every patch that makes its way into the core software</li>
+	<li>Open feeds of every commit that goes into the core software, including WordPress Trac's firehose and timeline</li>
```

Comment:
Should we mention GitHub here for Gutenberg and other externally maintained libraries, such as WordPress/Requests?

Reply:
If there is a limited number of additional places, then let's add them. If it's lengthy, maybe refer to them as a defined group? "and other externally maintained libraries used by WordPress" if that's an accurate statement.

Reply:
Technically "every commit that goes into the core software" is accurate because Gutenberg and Requests are committed directly as opposed to being bundled via Composer. I'm happy with this for now, I think any adjustments could be done in a follow up PR.